packages/tools/devtools/devtools-example/jest-puppeteer.config.js

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	server: {
+		command: `npm run start:client -- --port ${process.env["PORT"]}`,
+		port: process.env["PORT"],
+		launchTimeout: 10_000,
+		usedPortAction: "error",
+	},
+	launch: {
+		args: [
+			// https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#setting-up-chrome-linux-sandbox
+			"--no-sandbox",
+			"--disable-setuid-sandbox",
+
+			// Ensure our extension is loaded into the browser environment
+			"--load-extension=./dist/bundle",
```

Comment:
This package doesn't have access to the extension, this shouldn't be necessary here.

Reply:
Removed